### PR TITLE
Mle dev 1

### LINKED ISSUE #1: Adding code to github repo
Adding code to github repo

### PR DESCRIPTION
**Pull request by ankurkumar309**
added all the details in this branch. Merging it to master branch.

### CODE REVIEW

**Comment by ankurkumar309** [comment]
everything looks good

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.hiding
```
Comment (SharmaSubham975) [inline .gitignore:1]: Can exclude more files like ``csv, pycache etc``

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+ - Random Forest
+
+## Steps performed
+ - We prepare and clean the data. We check and impute for missing values.
+ - Features are generated and the variables are checked for correlation.
+ - Multiple sampling techinuqies are evaluated. The data set is split into train and test.
+ - All the above said modelling techniques are tried and evaluated. The final metric used to evaluate is mean squared error.
+
+## To excute the script
+python < scriptname.py >
```
Comment (SharmaSubham975) [inline README.md:18]: you mad add other description as well like ``how to activate environment``, ``how to execute script``, ``How to create environment``

File: env.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  - pip=20.2.4=py37h06a4308_0
+  - python=3.7.9=h7579374_0
+  - readline=8.0=h7b6447c_0
+  - setuptools=50.3.1=py37h06a4308_1
+  - sqlite=3.33.0=h62c20be_0
+  - tk=8.6.10=hbc83047_0
+  - wheel=0.35.1=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
+prefix: /home/tiger0537/miniconda3/envs/mle-dev
```
Comment (SharmaSubham975) [inline env.yml:24]: Try removing the prefix from ``env.yml`` file. You can make use of ``conda env export | grep -v "^prefix" > env.yml``.
Also, try to run the ``nonstandardcode.py`` file and resolve any error( ``Dependencies``) that occurs and retake the export using above mentioned command.

**Review by SharmaSubham975 — CHANGES_REQUESTED** [review]
Please address the following changes in the next review:

- Branch name can be ``enh/issue#1/base_code_commit``.
- Issue needs to be more elaborated something like ``Adding code to github repo``, it should explain the purpose of the task. Kindly refer [``4.3.3. Advice for the developer``](https://tigeranalytics-code-templates.readthedocs-hosted.com/en/latest/developer_guide/coding_best_practices.html)
- Create a PR with a ``meaningful title and a good description``. The title should clearly convey the intent of the PR.
- Remove prefix from ``env.yml`` file.

Thanks,

**Pull request closed without merging**